proxy/src/telemetry/metrics/http.rs

```diff
@@ -16,15 +16,15 @@ pub(super) type RequestScopes = Scopes<RequestLabels, Stamped<RequestMetrics>>;
 
 #[derive(Debug, Default)]
 pub(super) struct RequestMetrics {
-    total: Counter,
+    pub(super) total: Counter,
 }
 
 pub(super) type ResponseScopes = Scopes<ResponseLabels, Stamped<ResponseMetrics>>;
 
 #[derive(Debug, Default)]
 pub struct ResponseMetrics {
-    total: Counter,
-    latency: Histogram<latency::Ms>,
+    pub(super) total: Counter,
+    pub(super) latency: Histogram<latency::Ms>,
```

Comment:
similarly, it would be nice if we didn't have to expose these internals outside of tests

Reply:
Yeah, I'll add test-only accessors.